include/Telemetry/Telemetry.h

```diff
@@ -9,6 +11,68 @@
 #include "Arduino.h"
 #include <avr/pgmspace.h>
 
+// Set the protocall here: Serial, Ethernet
+#define Protocall Ethernet_TCP
```

Comment:
Spelling error: protocall -> protocol

Also, this helper stuff generally looks pretty reasonable, although I would request that macros/defines are named as ALL_CAPS, just as a style thing.